types/three/examples/jsm/webxr/XREstimatedLight.d.ts

```diff
@@ -0,0 +1,32 @@
+import { DirectionalLight, Group, LightProbe, XRFrame, WebGLRenderer, Texture } from '../../../src/Three';
+
+export class SessionLightProbe<TLight extends THREE.Group = THREE.Group> {
+    xrLight: TLight;
+    renderer: WebGLRenderer;
+    lightProbe: unknown;
+    xrWebGLBinding: unknown | null;
+    estimationStartCallback: () => void;
+    frameCallback: (time: number, xrFrame: XRFrame) => void;
+
+    constructor(
+        xrLight: TLight,
+        renderer: WebGLRenderer,
+        lightProbe: unknown,
+        environmentEstimation: boolean,
+        estimationStartCallback: () => void,
+    );
+
+    updateReflection: () => void;
```

Comment:
nit: I don't think it makes a difference, but I like to match the syntax between the JS and TS (here and in the rest of the PR). Ignore this comment if you want to.
```suggestion
    updateReflection(): void;
```